traittypes/tests/test_traittypes.py

```diff
@@ -17,7 +17,7 @@
 
 
 class IntArrayTrait(HasTraits):
-    value = Array().tag(dtype=np.int)
+    value = Array(dtype=np.int).tag(dtype=np.int)
```

Comment:
It seems the `dtype` arg was placed in the wrong spot. Well spotted! We should probably also remove the incorrect one.
```suggestion
    value = Array(dtype=np.int).tag()
```

Reply:
ok just completely remove the `.tag()` method call.

Reply:
Good point, updated the suggestion.